ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -400,7 +415,27 @@ internal class PaywallViewModelImpl(
     }
 
     private fun getCurrentLocaleList(): LocaleListCompat {
-        return LocaleListCompat.getDefault()
+        val preferredLocale = purchases.preferredUILocaleOverride
+        Logger.d("PaywallViewModel getCurrentLocaleList: preferredLocale = $preferredLocale")
```

Comment:
I think many of these logs should be verbose instead of debug... Wdyt? I would maybe only leave one as debug when "changing" locales.